### DISCUSSION TITLE
How to optimise the external scripts

### DISCUSSION THREAD

**Discussion by sarthak090** [vercel/next.js#50896] [Help]
### Summary

I am using next js and i have built the site with the help of wordpress rest api it was previously on wordpress , when i built it and tested it in pagespeed i got scores above 90+ but when i. Added google Analytics, google adesense , and other ads to the site the page speed score has reached 20 it is way lower than what it was on wordpress i have tried almost all the method like dynamic import delaying the script , lazy loading with next/scripts nothing seems to work

**Comment by icyJoseph** [comment]
Have you looked into https://partytown.builder.io/, it is supported by next/script AFAIK: https://nextjs.org/docs/app/api-reference/components/script#worker

**Reply by sarthak090** [reply]
I tried the worker strategy in next script is this different from that
